src/templates/color-extension.js

```diff
@@ -16,7 +17,9 @@ ${
     ${colors.map(color => `static let ${color.name} = ${
       extensionOptions.useCustomColorInitializer
         ? customColorTemplate(color)
-        : colorTemplate(color)
+        : extensionOptions.useColorLiterals
+          ? literalColorTemplate(color)
+          : colorTemplate(color)
```

Comment:
Honestly, I don't like this combination of two ternary operators. I guess we can replace it with `Picker` option with three states. What do you think?

Reply:
I think you're right — it's not the most readable. `Picker` is probably a better option as the three options we've got at the moment can't all be used together. If we swap how would you expect to see [color-extension.js](https://github.com/artemnovichkov/zepcode/blob/master/src/templates/color-extension.jsrl) formatted?

Reply:
I don't know how the picker option works in Zeplin. Anyway, if-else of switch constructions will be a better choice.

Reply:
Cool. I'll have a go at it and add it to this pull request.